server/utils/types/id_set.go

```diff
@@ -42,7 +42,7 @@ func NewIDSet(vv ...ID) *IDSet {
 	return i
 }
 
-func (i *IDSet) Set(v ID) {
+func (i *IDSet) Set(v ID) { //nolint:interfacer
```

Comment:
Maybe we just remove `interfacer` for `golangci.yml`? So far it's doesn't been useful.

Reply:
also agree, seems like the repo was even archived